### PR TITLE
Параничева А.В. 381806-1 ЛР №1

### PR DESCRIPTION
**Pull request by ParanichevaAlyona**
### Pull Request Readiness Checklist

See details at [code review]( https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/main/lab-guide/lab-1-setup.md#%D1%80%D0%B5%D1%86%D0%B5%D0%BD%D0%B7%D0%B8%D1%8F-%D0%BA%D0%BE%D0%B4%D0%B0)

- [x] Code is well documented and more information can be seen on the [website](https://devtools-course-practice-2021.readthedocs.io/en/latest/modules.html) (recommended)
- [x] Make sure that the testing has passed successfully on [GitHub Actions](https://github.com/UNN-ITMM-Software/devtools-course-practice/actions)
- [x] Get two approve from your fellow students
- [x] Take a review from one of the administrators

### CODE REVIEW

File: modules/complex-number/test/test_paranicheva_alyona_complex_number.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#include <gtest/gtest.h>
+#include <math.h>
+
+#include "include/complex_number.h"
+
+TEST(Paranicheva_Alyona_ComplexNumberTest, Module) {
+    ComplexNumber z(6.0, -8.0);
+
+    double resmodule = sqrt(pow(z.getRe(), 2) + pow(z.getIm(), 2));
+    EXPECT_EQ(10.0, resmodule);
```
Comment (BelikJulia) [inline modules/complex-number/test/test_paranicheva_alyona_complex_number.cpp:12]: EXPECT_EQ() можно заменить на EXPECT_DOUBLE_EQ()

File: modules/complex-number/test/test_paranicheva_alyona_complex_number.cpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+#include <gtest/gtest.h>
+#include <math.h>
+
+#include "include/complex_number.h"
+
+TEST(Paranicheva_Alyona_ComplexNumberTest, Module) {
+    ComplexNumber z(6.0, -8.0);
+
+    double resmodule = sqrt(pow(z.getRe(), 2) + pow(z.getIm(), 2));
```
Comment (SMolotkova) [inline modules/complex-number/test/test_paranicheva_alyona_complex_number.cpp:11]: Можно заменить sqrt(pow, pow) на hypot()

**Review by SMolotkova — APPROVED** [review]
Отлично)

**Review by BelikJulia — APPROVED** [review]
Все хорошо

File: modules/complex-number/test/test_paranicheva_alyona_complex_number.cpp [hunk 3]
```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 Paranicheva Alyona
+
+#include <gtest/gtest.h>
+#include <cmath>
```
Comment (streltsova-yana) [inline modules/complex-number/test/test_paranicheva_alyona_complex_number.cpp:4]: Заголовочные файлы C и C++ лучше отделять пустой строкой
[See more details](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

**Review by streltsova-yana — CHANGES_REQUESTED** [review]

**Review by streltsova-yana — APPROVED** [review]

**Pull request merged**